rpa-bot/1.3/modules/ROOT/pages/configuration.adoc

```diff
@@ -55,19 +61,20 @@ Format: `+http://<Serveraddress>:<Port>+`, for example: `+http://myproxy.de:881+
 [[session-settings]]
 == Session Settings
 
-Enable the bot to use secure sessions by entering credentials the bot uses to log in to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+If you want to have a default user account, that the RPA Bot uses to log in to the session on the computer, where the bot runs processes, enter the credentials here. When users create a process run configuration in RPA Manager, they can choose this user or specify another there.
 
 image:bot-configurator-session-settings.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
 
-* *Default Secure Session User*
+* *Default Session User*
++
+If you want the bot to have a default user to log in to the session for the execution of a process, enter the credentials here. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`. For domain users, use the following form: `mydomain\username`.
```

Comment:
Can you use similar terminology as on line 64?

To specify that the RPA Bot use a default user account to log in to the session and run processes on the computer, enter the credentials here.